packages/types/src/srml/parachains/types.ts

```diff
@@ -0,0 +1,167 @@
+// Auto-generated, do not edit
+
+import { Enum, Struct, Tuple, Vector } from '../../codec';
+import { AccountId, Bytes, Hash, Signature, U32, u64 } from '../../primitive';
```

Comment:
Only allowed imports are from codec and primitives. (There needs to be a way to link across modules as well)